src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -252,7 +256,12 @@ def default_api_version(self):
         ResourceType.MGMT_APPSERVICE: '2018-02-01',
         ResourceType.MGMT_EVENTHUB: '2018-01-01-preview',
         ResourceType.MGMT_IOTHUB: '2019-07-01-preview',
-        ResourceType.MGMT_DATABOXEDGE: '2019-08-01'
+        ResourceType.MGMT_DATABOXEDGE: '2019-08-01',
+        ResourceType.MGMT_CONTAINERREGISTRY: '2019-05-01',
+        ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2020-11-01', {
```

Comment:
why do you specify different api version for this?

Reply:
this will bring in the "profile" support effort for AKS team.
because we were always testing/release the freshest api version.

Reply:
In Azure Stack environment, we may not use the latest API version. Here is for 2020-09-01-hybrid profile. @Bhuvaneswari-Santharam for confirmation

Reply:
Yes, these are for the 2020-09-01-hybrid profile, and I have another PR to add test for hybrid profiles